src/Core/src/Platform/Windows/ViewExtensions.cs

```diff
@@ -361,39 +359,11 @@ internal static void UnfocusControl(Control control)
 			if (control == null || !control.IsEnabled)
 				return;
 
-			// "Unfocusing" doesn't really make sense on Windows; for accessibility reasons,
-			// something always has focus. So forcing the unfocusing of a control would normally 
-			// just move focus to the next control, or leave it on the current control if no other
-			// focus targets are available. This is what happens if you use the "disable/enable"
-			// hack. What we *can* do is set the focus to the Page which contains Control;
-			// this will cause Control to lose focus without shifting focus to, say, the next Entry 
-
-			if (ContainingPage == null)
-			{
-				// Work our way up the tree to find the containing Page
-				DependencyObject parent = control;
-
-				while (parent != null && parent is not Page)
-				{
-					parent = VisualTreeHelper.GetParent(parent);
-				}
-
-				ContainingPage = parent as Page;
-			}
-
-			if (ContainingPage != null)
-			{
-				// Cache the tabstop setting
-				var wasTabStop = ContainingPage.IsTabStop;
-
-				// Controls can only get focus if they're a tabstop
-				ContainingPage.IsTabStop = true;
-				ContainingPage.Focus(FocusState.Programmatic);
-
-				// Restore the tabstop setting; that may cause the Page to lose focus,
-				// but it won't restore the focus to Control
-				ContainingPage.IsTabStop = wasTabStop;
-			}
+			var isTabStop = control.IsTabStop;
+			control.IsTabStop = false;
+			control.IsEnabled = false;
```

Comment:
@PureWeen @rmarinho What do these lines serve to do if we're resetting them in the following 2 lines?

Reply:
They actually make the control to unfocus (disabling it)